app/views/studies/_initialize_expression_form.html.erb

```diff
@@ -102,7 +102,8 @@
       function handleRawCountsUpdate() {
         <% current_values = study_file.associated_matrix_files(:raw)&.map { |sf| { label: sf.name, value: sf.id.to_s } } %>
         const currentValues = <%= current_values.to_json.html_safe %>
-        updateRawCountsAssnSelect(expFormId_<%= allow_only %>, currentValues)
+        const isRequired = <%= User.feature_flag_for_instance(current_user, 'raw_counts_required_backend') %>
+        updateRawCountsAssnSelect(expFormId_<%= allow_only %>, currentValues, isRequired)
```

Comment:
The issue precedes this PR, but interpolating variable _names_ seems even less easily maintainable / debuggable than interpolating variable values.

Reply:
Again - totally agreed, but with the current infrastructure on this page, it's the only thing that works.  Otherwise everything errors out, and you can't even upload files due to numerous JS errors.  Really the only solution is the large-scale refactoring we've been discussing.